brambling/models.py

```diff
@@ -1232,13 +1234,18 @@ def refund(self, amount=None, bought_items=None, issuer=None, dwolla_pin=None):
         if bought_items is None:
             bought_items = self.bought_items.all()
 
-        total_refunds = self.related_transaction_set.aggregate(Sum('amount'))['amount__sum'] or 0
-        refundable = self.amount + total_refunds
-        if refundable < amount:
-            raise ValueError("Not enough money available")
+        total_refunds = self.related_transaction_set.aggregate(Sum('amount'))['amount__sum']
+        if total_refunds is not None:
+            refundable = self.amount + total_refunds
+            if refundable == 0:
+                return None
 
-        if refundable == 0:
-            return None
+            # SB: Pretty sure this just disables completing partial
+            # refunds.
+            if refundable < amount:
+                raise ValueError("Not enough money available")
```

Comment:
Refundable is the amount that hasn't been refunded yet. (total_refunds is negative.) So it will never be more than amount. It'll be less if a refund of any kind has already happened.